packages/components/src/core-badge/style.module.scss

```diff
@@ -0,0 +1,54 @@
+@use '@wordpress/base-styles/colors';
+@use '@wordpress/base-styles/mixins';
+@use '@wordpress/base-styles/variables';
+@use '../utils/_theme-variables' as theme;
+
+$badge-colors: (
+	"info": theme.$components-color-accent,
+	"warning": colors.$alert-yellow,
+	"error": colors.$alert-red,
+	"success": colors.$alert-green,
+);
+
+.components-badge {
```

Comment:
Since these are modules, I think we should keep the names very light, like ".badge", ".icon", ".content"

Reply:
We [discussed](https://github.com/Automattic/wp-calypso/pull/103004/files#r2068564738) this a bit in a previous attempt, and I'm currently leaning towards maintaining the original naming conventions. Easier search, easier to convert existing stylesheets, less context switching across mixed codebases, no need to change the coding guidelines, same mental model (BEM) for hygiene, etc. Thoughts?

Reply:
The classes in the current file are fine to me. But I think some of the current guidelines result in way too big classnames and it might not be needed for CSS modules.

>  Easier search

Is that really important given that you actually "import" these, so you know exactly what's used where.

---

I guess I'm not sure either way at this point :)

Reply:
I'm not sure either, we're in new territory 🙂 I guess we can at least drop the `components-` prefix, that's for sure.

Reply:
I'd personally agree that we could at least avoid the package-level namespacing prefix (`components-`), and that BEM conventions around children selectors could still persist to have an easy, predictable way of naming child components. I think we'll want to mention this in our CSS modules guidelines.

Reply:
Done in 421a06a